### PR TITLE
Laravel 10 upgrade 

### PR DESCRIPTION
**Pull request by DominicV1**
Compatible with laravel10

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,21 +17,20 @@
...
         "sendinblue/api-v3-sdk": "^8.3",
         "spatie/laravel-package-tools": "^1.9.2",
         "symfony/http-client": "^6.1",
         "symfony/sendinblue-mailer": "^6.1"
     },
     "require-dev": {
         "laravel/pint": "^1.0",
-        "nunomaduro/collision": "^6.0",
-        "orchestra/testbench": "^7.0",
+        "nunomaduro/collision": "6.1",
```
Comment (jhhazelaar) [inline composer.json]: Please don't use fixed version numbers.

File: composer.json [hunk 2]
```diff
@@ -17,21 +17,20 @@
...
+        "orchestra/testbench": "^8.0",
         "pestphp/pest": "^1.21",
         "pestphp/pest-plugin-laravel": "^1.1",
-        "phpunit/phpunit": "^9.5"
+        "phpunit/phpunit": "9.6"
```
Comment (jhhazelaar) [inline composer.json]: Please don't use fixed version numbers.

File: composer.json [hunk 3]
```diff
@@ -17,21 +17,20 @@
     ],
     "require": {
         "php": "^8.1",
-        "illuminate/contracts": "^9.0",
-        "illuminate/mail": "^9.27",
-        "illuminate/support": "^9.27",
+        "illuminate/contracts": "*",
```
Comment (jhhazelaar) [inline composer.json]: Please use a version number.

File: composer.json [hunk 4]
```diff
@@ -17,21 +17,20 @@
     ],
     "require": {
         "php": "^8.1",
-        "illuminate/contracts": "^9.0",
-        "illuminate/mail": "^9.27",
-        "illuminate/support": "^9.27",
```
Comment (jhhazelaar) [inline composer.json:22]: Where did support go?

Comment (DominicV1) [reply composer.json:22]: Uhm, good question

**Pull request merged**